etl/steps/data/garden/un/2024-04-09/undp_hdr.meta.yml

```diff
@@ -404,9 +406,9 @@ tables:
         unit: international-$ in 2017 prices
         description_short: |-
           Measures the total income earned by residents of a country, including income earned abroad. This data is adjusted for inflation and differences in the cost of living between countries.
-        description_key: &description_key_gnipc
-          - GNI per capita (formerly GNP per capita) is the gross national income, converted to U.S. dollars using the World Bank Atlas method, divided by the midyear population. GNI is the sum of value added by all resident producers plus any product taxes (less subsidies) not included in the valuation of output plus net receipts of primary income (compensation of employees and property income) from abroad (World Bank).
-          - GNI, calculated in national currency, is usually converted to U.S. dollars at official exchange rates for comparisons across economies, although an alternative rate is used when the official exchange rate is judged to diverge by an exceptionally large margin from the rate actually applied in international transactions (World Bank).
+        description_key:
+          - &description_key_gnipc1 GNI per capita (formerly GNP per capita) is the gross national income, converted to U.S. dollars using the World Bank Atlas method, divided by the midyear population. GNI is the sum of value added by all resident producers plus any product taxes (less subsidies) not included in the valuation of output plus net receipts of primary income (compensation of employees and property income) from abroad (World Bank).
```

Comment:
I missed this description and it's not quite for the indicator UNDP uses (they use PPP rates). I guess it's better to rephrase page 4 of the technical notes (and copy it in description_from_producer)